app/helpers/application_helper.rb

```diff
@@ -568,46 +568,57 @@ def model_report_type(model)
     nil
   end
 
+  def show_taskbar_in_header?
+    return false if @explorer
+    return false if controller.action_name.end_with?("tagging_edit")
+
+    hide_actions = %w(
```

Comment:
Any reason this isn't a constant?  Seems weird that we are instantiating this every time we run this method.